kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RollingUpdateProcessServiceIntegrationTest.java

```diff
@@ -254,4 +275,178 @@ public void testStartProcessInDifferentDeploymentWithContainerId() throws Except
         taskClient.completeAutoProgress(CONTAINER_ALIAS, task.getId(), USER_YODA, null);
 
     }
+
+    @Test
+    public void testDifferentGAVsWithAlias() throws Exception {
+        Long oldPid = processClient.startProcess(CONTAINER_ALIAS, PROCESS_ID_EVALUATION);
+        assertNotNull(oldPid);
+        assertTrue(oldPid > 0);
```

Comment:
Not sure what information about the value will `assertTrue` give you in case it's not greater than 0. If you care about the value, there are assertions for numbers. 
And in general, using AssertJ simplifies assertions in tests. Have you considered using the library?